modules/uavcan/canard_dsdlc/canard_dsdlc_helpers.py

```diff
@@ -1,7 +1,9 @@
+#!/usr/bin/env python3
+
 import os
-import uavcan
+import pyuavcan_v0 as uavcan
 import errno
-import em
+import em   # NOTE: must use empy v3.3.4
```

Comment:
Is there a way to specify this version is imported? What is the reason to require 3.3.4 specifically?

Reply:
I've found ways to install a specific version of a library on your system but not a way to import a specific verison.

We make a call to an empy function called expand().  The way we call the function conforms to v3.3.4.  It's not clear to me (based on the documentation) on how the call should change for v4.2.  I've tried different things based on the [not-so-clear] documentation and examples I've found.  But we don't get the desired output.

Reply:
No problem. I was just curious